properties_pane/entity_level/entityLevelConfig.json

```diff
@@ -574,7 +574,27 @@ making sure that you maintain a proper JSON format.
 						"propertyName": "Name",
 						"propertyKeyword": "indxName",
 						"propertyTooltip": "Optional, if not specified an automatic name will be assigned. Index name are needed to drop indexes and appear in error messages when a constraint is violated.",
-						"propertyType": "text"
+						"propertyType": "text",
+						"dependency": {
+							"type": "not",
+							"values": {
+								"key": "ifNotExist",
+								"value": true
+							}
+						}
+					},
+					{
+						"propertyName": "Name",
+						"propertyKeyword": "indxName",
+						"propertyTooltip": "",
```

Comment:
Where did the [original tooltip](https://github.com/hackolade/CockroachDB/pull/35/files#diff-51852b2e184769a14b98d38122cf620c9644fbbfd6daec8f4e45a384855b931aR576) goes?

Reply:
This tooltip makes no sense since the property is required in particular use case